.circleci/config.yml

```diff
@@ -0,0 +1,102 @@
+####################
+# CircleCI configuration reference:
+#   https://circleci.com/docs/2.0/configuration-reference
+####################
+# CircleCI built-in environment variables:
+#   https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables
+####################
+
+
+####################
+# Templates: see "anchors" in https://learnxinyminutes.com/docs/yaml/
+####################
+
+common_settings: &common_settings
+  docker:
+    - image: python:2.7
+  working_directory: ~/python_mozetl
+
+install_dependencies: &install_dependencies
+  name: install dependencies
+  command: |
+    apt update
+    apt install -y libsnappy-dev default-jre
```

Comment:
I'd prefer we use  openjdk-8-jre-headless here so that we aren't surprised by a new java version.